sm_c/enclave.c

```diff
@@ -108,13 +128,21 @@ static inline void context_switch_to_host(uintptr_t* encl_regs,
   swap_prev_state(&enclaves[eid].threads[0], encl_regs);
   swap_prev_mepc(&enclaves[eid].threads[0], read_csr(mepc));
 
+  switch_vector_host(); 
+
+  uintptr_t interrupts = MIP_SSIP | MIP_STIP | MIP_SEIP;
+  write_csr(mideleg, interrupts);
+
   // enable timer interrupt
-  set_csr(mie, MIP_MTIP);
+  clear_csr(mie, MIP_MTIP);
```

Comment:
Why are we disabling the timer interrupt here?

Reply:
Whoops! This was a remnant of my old code. Removing...